source/utils/RandomUtils.cc

```diff
@@ -0,0 +1,42 @@
+// ----------------------------------------------------------------------------
+// nexus | RandomUtils.cc
+//
+// This class defines commonly used functions to generate random quantities.
+//
+// The NEXT Collaboration
+// ----------------------------------------------------------------------------
+#include "RandomUtils.h"
+
+#include <Randomize.hh>
+
+#include "CLHEP/Units/SystemOfUnits.h"
+
+namespace nexus {
+
+  G4double UniformRandomInRange(G4double max_value,G4double min_value)
```

Comment:
Please add the missing space in the parameter list. I think that similar minor problems like this remain in other places, but it's probably not reasonable to point them all out, here.